### ISSUE TITLE
Can you train STANet? When I train it,Imeet a error.

### ISSUE THREAD

**Issue by wangjin1234567891** [likyoo/change_detection.pytorch#3]
![image](https://user-images.githubusercontent.com/76192707/132642598-e19fa795-b152-4aae-9088-35f80c719c01.png)

**Comment by likyoo** [comment]
@LinoSun

**Comment by LinoSun** [comment]
Could plz show more code.

**Comment by wangjin1234567891** [comment]
I trian Unet and it's ok .I just change the model as following picture shows 
![image](https://user-images.githubusercontent.com/76192707/132660500-0cbe6c40-f33b-4030-8818-af2c04d48755.png)

**Comment by LinoSun** [comment]
We still need debug STANet later.If you want use it now. You can try to use BCLLoss or other loss function for STANet because of STANet model only forward one channel distance map.

**Comment by wangjin1234567891** [comment]
Ok, thans a lot.
